opba-protocols/xs2a-protocol-tests/xs2a-bdd-tests-common/src/main/java/de/adorsys/opba/protocol/xs2a/tests/e2e/stages/AccountInformationResult.java

```diff
@@ -322,11 +321,10 @@ public AccountInformationResult user_anton_brueckner_provided_initial_parameters
     }
 
     public AccountInformationResult fintech_calls_consent_activation_for_current_authorization_id() {
-        RestAssured
+        withSignatureHeaders(RestAssured
```

Comment:
Note, here it is OK - called by FinTech

Reply:
Removed signature header adding for non-fintech requests